src/pydvl/value/sampler.py

```diff
@@ -205,6 +200,24 @@ def weight(cls, n: int, subset_len: int) -> float:
         ...
 
 
+class StochasticSampler(PowersetSampler[T], abc.ABC):
```

Comment:
I think this is clear case for a Mixin, to avoid an additional level in the class hierarchy.